### PR TITLE
[TTP] Move set up tap to pay to a separate section and show request for feedback if its been used before

### LINKED ISSUE #8458: [Mobile Payments] Move Set up Tap To Pay to a separate section and show request for feedback if it's been used before
Image

### PR DESCRIPTION
**Pull request by kidinov**
Closes: #8458

### Description

Design UsEjcaCj3qJ5pqnXeYz7dq-fi-329%3A19231

The PR:
* Adjusts the UI to match the design
* Adds request for feedback if TTP was used at least once
The request for feedback button does nothing so far

### Testing instructions

* Open the Payments hub and notice that request for TTP feedback is missing
* Run any TTP flow (testing/simple payment/order)
* Open the Payments hub and notice that request for TTP feedback is there

### Images/gif

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WooCommerce/src/main/res/layout/fragment_card_reader_hub.xml [hunk 1]
```diff
@@ -1,70 +1,68 @@
...
+                android:layout_width="match_parent"
+                android:layout_height="0dp"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@id/cardReaderHubLoading"
+                app:layout_constraintVertical_chainStyle="spread_inside"
+                tools:itemCount="10"
+                tools:listitem="@layout/card_reader_hub_list_item" />
+
+            <!-- android:textColorHighlight="#00FFFFFF": Workaround for a bug https://stackoverflow.com/a/70394538/632516 -->
```
Comment (AnirudhBhat) [inline WooCommerce/src/main/res/layout/fragment_card_reader_hub.xml]: ❓ The comment here doesn't make sense. I believe it should be above the `cardReaderHubOnboardingFailedTv` view. WDYT?

**Review by AnirudhBhat — APPROVED** [review]
Thanks for your work on this @kidinov. It works as expected. I've left one minor comment that is non-blocking for the merge.

I do have a couple of clarifying questions. Again, nothing is blocking the merge.

1. It seems like we consider the click on "Tap To Pay" in collect payment screen to display the feedback. Can we push this logic a little further down the process? I mean, can we display the TTP feedback only if the merchant connects to the built-in reader or processes a payment using TTP? My thinking is that the feedback will be more effective if we push the TTP-used logic further down the line.

2. The text in the Try a payment screen says `You can find Tap to Pay on Android in Menu > Payments`. Is this correct? I mean, the merchant finds the TTP tutorial in Menu > Payments correct? The TTP feature is found in the collect payment screen, right?

3. Related to the text above. `You can find Tap to Pay on Android in Menu > Payments`. Do we need to mention the word "Android" there? it feels more like a distraction since we could just say `You can find Tap to Pay in Menu > Payments`. If that text is coming locally as opposed to from the server, I don't see the usefulness of adding an "Android" word there since the users will be seeing this on the Android device anyway.

4. Again, related to the above point. Once we identify that TTP has been used. We change the text in the payments menu to "Use your phone to accept card payments". It sounds like this is an entry point to use the TTP feature. However, it is actually just a tutorial that helps us how to use TTP. Isn't this misleading?

**Comment by kidinov** [comment]
@AnirudhBhat huge thanks for feedback and review!

> It seems like we consider the click on "Tap To Pay" in collect payment screen to display the feedback. Can we push this logic a little further down the process? I mean, can we display the TTP feedback only if the merchant connects to the built-in reader or processes a payment using TTP? My thinking is that the feedback will be more effective if we push the TTP-used logic further down the line.

I had precisely the same thoughts! Ended up with this solution for the following reasons:
* If people have nothing to give feedback about then most of them won't do it
* We may lose people who will have troubles at the beginning of the process, like you mentioned they may not be able to connect to a reader at all and these people still need (and actually probably most valuable) to collect feedback from
Wdyt?

> The text in the Try a payment screen says You can find Tap to Pay on Android in Menu > Payments. Is this correct? I mean, the merchant finds the TTP tutorial in Menu > Payments correct? The TTP feature is found in the collect payment screen, right?

This has been proposed and validated by Ann so I assume this is ok. Both collect and try a payment flow starts from Menu -> Payments so I guess that's the idea to head people in this direction

> Related to the text above. You can find Tap to Pay on Android in Menu > Payments. Do we need to mention the word "Android" there? it feels more like a distraction since we could just say You can find Tap to Pay in Menu > Payments. If that text is coming locally as opposed to from the server, I don't see the usefulness of adding an "Android" word there since the users will be seeing this on the Android device anyway.

Yeah, this sounds weird to me as well, but that's how they call it https://android-developers.googleblog.com/2023/02/how-stripe-leveraged-google-play-to-build-an-sdk-for-tap-to-pay-on-android.html so probably it's fine to leave it as it is


> Again, related to the above point. Once we identify that TTP has been used. We change the text in the payments menu to "Use your phone to accept card payments". It sounds like this is an entry point to use the TTP feature. However, it is actually just a tutorial that helps us how to use TTP. Isn't this misleading?

I asked Ann if we can improve that somehow

File: WooCommerce/src/main/res/layout/fragment_card_reader_hub.xml (hunk 1 above)
Comment (kidinov) [reply WooCommerce/src/main/res/layout/fragment_card_reader_hub.xml]: That's the correct place but this is redundant. The same comment is inside the included layout. I removed that

**Pull request merged**